content/Hardware Support/Opta/Recover-an-Arduino-Opta-with-read-only-flash-memory.md

```diff
@@ -0,0 +1,41 @@
+---
+title: "Recover an Arduino Opta with read-only flash memory"
+---
+
+In certain situations, the QSPI flash memory on the Arduino Opta may become configured as read-only.
+
+The QSPI flash is likely read-only if:
+
+* You are unable to erase or overwrite the QSPI flash memory.
+* You encounter an `Error formatting OTA partition` message when attempting to [partitioning the memory](https://support.arduino.cc/hc/en-us/articles/16206977438748-Reset-the-flash-memory-on-STM32H747-based-devices).
+
+The write protection can be removed by uploading a recovery sketch.
+
+----
+
+<!--  -->
+
+Follow these steps:
+
+1. Download or copy the [unlockOptaNORFlash.ino](https://github.com/manchoz/unbrick-recover-readonly-flash-arduino-opta/blob/master/unlockOptaNORFlash/unlockOptaNORFlash.ino)[^1] sketch from GitHub.
+
+   ![Downloading a file on GitHub.](img/github-download.png)
+
+1. Open the sketch in your editor:
+
+   * Arduino IDE: Do one of the following:
```

Comment:
```suggestion
   * In Arduino IDE, do one of the following:
```
Maybe this format could be used to avoid using " : " twice on the same sentence.